apps/omg_watcher_info/lib/omg_watcher_info/db/eth_event.ex

```diff
@@ -237,23 +240,25 @@ defmodule OMG.WatcherInfo.DB.EthEvent do
           event
       end
 
-    tx_output = resolve_tx_output(output_pointer)
+    case resolve_tx_output(output_pointer) do
+      %DB.TxOutput{} = tx_output ->
+        :ok = insert_exit_if_not_exist(ethevent, tx_output)
 
-    insert_exit_if_not_exist(ethevent, tx_output)
+      # it dies when output is expected to exist (most of the cases, see: `elixir-omg/issues/1760`)
```

Comment:
what is `most of the cases`?

Reply:
I've added better description with a link to the original issue.